README.md

```diff
@@ -1054,6 +1054,7 @@ Table of Contents
   * [huaweicloud.com](https://www.huaweicloud.com/intl/en-us/product/dns.html) – Free DNS hosting by Huawei
   * [Hetzner](https://www.hetzner.com/dns-console) – Free DNS hosting from Hetzner with API support.
   * [Glauca](https://docs.glauca.digital/hexdns/) – Free DNS hosting for up to 3 domains and DNSSEC support
+  * [Nova53](https://www.nova53.net) - Free Anycast DNS hosting for up to 3 domains, DNSSEC support, Master Slave and Native support.
```

Comment:
Not sure if it’s just me but their site seems a bit broken. Doesn’t load completely, sig up link just goes nowhere etc. 

Also best to avoid the master and slave terminology - rather use what’s on their site.

Reply:
the site loads fine for me, and that's the terminology for it
![](https://cdn.darrennathanael.com/assets/github/FFE5E9F0-21DC-479E-B812-25B4A779020A.jpeg)

Reply:
maybe "Primary Secondary and Native DNS Zone support." would fit better?

Reply:
Yes, their landing uses primary and secondary.

Reply:
Taking a huge time to load for me also. no site icon.

Reply:
It's failing because the admin interface is on group53.com and thats blocked in most ad blockers.

Just seems like trouble to me.

Reply:
the domain group53.com is expired, hence its blocked on ADB's.
Lemme contact Darren on discord(i know him there)